src/web/layouts/StandardLayout.tsx

```diff
@@ -393,16 +399,17 @@ export const StandardLayout = ({
                     )}
 
                     <Section
+                        data-print-layout='hide'
                         backgroundColour={background.primary}
                         padded={false}
                         showTopBorder={false}
                     >
-                        <GuardianLines count={4} pillar={pillar} />
+                        <GuardianLines data-print-layout='hide' count={4} pillar={pillar} />
```

Comment:
Again, nested above? There's a few more nested ones, won't comment on all of them, but surprised we need them